src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -106,6 +106,12 @@ def is_update_needed(output_folder: str, update_on_events_update: bool, client:
             os.remove(destination_event_file_path)
             latest_event_tp.close()
             need_update = True
+    # if the logs info in the cluster metadata isn't set to final status we should update
+    cluster_md = get_cluster_metadata(output_folder)
+    with open (cluster_md) as f:
+        md = json.load(f)
+        if md.get("logs_info", "") not in ("timeout", "completed", ""):
+            need_update = True
```

Comment:
nit: add a blank line for having a breath when reading the code :)

Reply:
Now that the open and json.load moved elsewhere it should be easier to read.